src/datastructures.apib

```diff
@@ -20,6 +20,9 @@
 + postal_code: `9000` (string, required, nullable)
 + city: `Ghent` (string, required, nullable)
 + country: `BE` (string, required)
++ area_level_two (object, nullable)
```

Comment:
Should it be `nullable` and always show. 
Or `optional` and only be given when set?

Reply:
My pref is nullable and always shown.

Reply:
> My pref is nullable and always shown.

Same.

Reply:
Asked because it's already a pattern on the address.
`addressee` under `Addressee` is already only shown when set. 
As a PP, I do prefer `nullable` too.